src/routes/Apps/AppsListComponent.tsx

```diff
@@ -313,6 +308,25 @@ export class Component extends React.Component<Props, ComponentState> {
             </div>
         </div>
     }
+
+    private getSortedApplications(sortColumn: ISortableRenderableColumn, apps: CLM.AppBase[]): CLM.AppBase[] {
```

Comment:
Why was this moved? The whole ordering of public, private stuff seems like a weaker convention to follow than the stronger convention of having `render` be the last method in `React`. Not everyone is using TypeScript or private. Moving code is harder for code review / diffs as you can't tell was it moved and changed for just moved.

Reply:
If we make that choice we should add an exclusion rule is TSLINT so that it stops showing errors